crates/plugins/src/prompt.rs

```diff
@@ -0,0 +1,44 @@
+use crate::manifest::{PluginManifest, PluginPackage};
+use anyhow::Result;
+use std::io;
+
+/// Prompts user as to whether they trust the source of the plugin and
+/// want to proceed with installation.
+fn print_prompt(name: &str, license: &str, source_url: &str) {
```

Comment:
Suggest either renaming this to something like "print_confirm_install_prompt" or having the prompt text as a parameter - at the moment it is a very generic name for a very specific prompt.